servicenow_consts.py

```diff
@@ -41,19 +41,25 @@
 SERVICENOW_SUCC_CONNECTIVITY_TEST = "Test Connectivity Passed"
 SERVICENOW_ERR_SERVER_CONNECTION = "Connection failed"
 SERVICENOW_VALIDATE_INTEGER_MESSAGE = "Please provide a valid integer value in the {key} parameter"
-SERVICENOW_ERR_FETCH_VALUE = 'Error occurred while fetching variable value for the item_option_value: {item_opt_value} of the System ID: {sys_id}'
-SERVICENOW_ERR_FETCH_QUESTION_ID = 'Error occurred while fetching question ID for the item_option_value: {item_opt_value} of the System ID: {sys_id}'
-SERVICENOW_ERR_FETCH_QUESTION = 'Error occurred while fetching question for the question ID: {question_id} and the item_option_value: {item_opt_value} of the System ID: {sys_id}'
+SERVICENOW_ERR_FETCH_VALUE = 'Error occurred while fetching variable value \
+    for the item_option_value: {item_opt_value} of the System ID: {sys_id}'
+SERVICENOW_ERR_FETCH_QUESTION_ID = 'Error occurred while fetching question ID for \
+    the item_option_value: {item_opt_value} of the System ID: {sys_id}'
```

Comment:
suggestion: we might be able to fit the whole text on the 2nd line with bracket (e.g. '(...)')

Reply:
Yes I have made the suggested changes.